src/app/weather-forecast/weather-forecast.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="weatherForecastSpace">
+
+    <h2>{{response.name}} weather forecast</h2>
+    <h3>{{((response.main.temp -273.15)-1) | number:'1.0-0'}}°</h3>
+    <h3>Pressure {{response.main.pressure}} torr.</h3>
+    <h3>{{response.weather[0].main}}</h3>
```

Comment:
what happen if the API changes its data model?
Try to map the API model tu an object model for this component